### DISCUSSION TITLE
Welcome to Sucrose Discussions!

### DISCUSSION THREAD

**Discussion by Taiizor** [Taiizor/Sucrose#4] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by n0tic** [comment]
Hello,

I recently downloaded and started using your application. I'm really enjoying the various wallpapers available; they're beautiful! However, I've noticed that the wallpapers only activate on one of my monitors, even though I have three. Strangely, they don't activate on my primary-set monitor.

Is this a feature you plan on adding support for in a future update or have I missed a settings option?
It's hard to understand as it is not in english right now.

Cheers, N~

**Reply by Taiizor** [reply]
Hello,

First of all, I'm glad that you're using the application and liking it. Multi-monitor support is indeed already present in the application's infrastructure, but the user interface part is not yet finished, so you can't use it right now. The interface will be ready very soon.

![Sucrose](https://github.com/Taiizor/Sucrose/assets/41683699/cf01dc72-1d3d-4a0a-8d9e-0e9f1d2ba821)

However, if you don't want to wait, I have shared three different configuration files that support multi-monitor setups. You can copy this configuration file named `Engine.json` to the directory `C:\Users\<UserName>\AppData\Roaming\Sucrose\Setting` and test it. If you want to revert the changes, you can simply delete the `Engine.json` configuration file.

[Setting.zip](https://github.com/Taiizor/Sucrose/files/12776916/Setting.zip)

To explain the configuration file:

When `"DisplayType": 0`, the `"ScreenIndex": 0` variable plays the wallpaper only on the monitor with index 0. You can change the value 0 to switch between monitors.

When `"DisplayType": 1`, it expands the wallpaper across all your monitors.

When `"DisplayType": 2`, it duplicates the wallpaper to each of your monitors.

You are correct about the translations. Since the application is still actively in development, I can't provide translations instantly as it can extend the process.

I hope this explanation is clear enough. If you have any questions or if there's anything you don't understand, please don't hesitate to reach out again.

**Reply by Taiizor** [reply]
The version with the translations was published today. You can experience it by updating Sucrose.

**Reply by Taiizor** [reply]
Display Preferences will be available in the next pre-release version.

![image](https://github.com/Taiizor/Sucrose/assets/41683699/1714c4b5-001a-4498-9e1f-bbb788d90cbb)

**Reply by Taiizor** [reply]
The new pre-release version has been released, you can experience the display preferences.

**Comment by anakin15a** [comment]
Bonjour j'ai 3 écran et je ne peut Que mettre un fond d'écran pour l'un de mais écran 3 écran alors que j'aimerai changer entre le fond d'écran du premier du deuxième et du troisième si c'est possible bien sur

**Reply by mericbozkurt** [reply]
Vous devriez peut-être utiliser Lively, qui dispose d'une telle fonctionnalité.

Sincèrement.

**Reply by Taiizor** [reply]
Il semble que vous vouliez définir des papiers peints différents pour chaque écran. Actuellement, l'infrastructure de l'application ne prend pas en charge cette fonctionnalité. Vous pouvez envisager des applications alternatives en tenant compte du commentaire ci-dessus.

Si je me trompe et que vous souhaitez mettre le même papier peint sur les 3 écrans, cette fonctionnalité ne fonctionne pas correctement pour le moment. Un problème lié à cela a été signalé dans le lien ci-dessous, où vous pouvez le suivre pour obtenir des informations sur la situation.

https://github.com/Taiizor/Sucrose/issues/81

**Comment by Yin-Vannthy** [comment]
<img width="1919" height="1194" alt="image" src="https://github.com/user-attachments/assets/bb038e11-65e7-4b5e-9b4e-b1df27dcf4ef" />

@Taiizor 
Excuse me
Why is the Duplicate Same Wallpaper option disabled?

**Reply by Taiizor** [reply]
When developing the Sucrose application, I initially implemented `Duplicate Same Wallpaper` through a truly optimized solution. However, as the application evolved, new features and engines were added. As a result, this previously optimized solution started causing issues—it stopped being a solution and became a problem. That’s why I’ve temporarily disabled this feature.

**Reply by Yin-Vannthy** [reply]
@Taiizor 
Thank you for your hard work.
May I ask, does the `Sucrose Wallpaper Engine` have a feature to add live wallpaper to the lock screen?
Do you have a plan to add a feature that allows users to add widgets(Date, Clock, etc.) to the wallpaper?

Currently, I am facing this issue with the clock wallpaper.
The wallpaper with the red download button cannot be used.
<img width="1916" height="1196" alt="image" src="https://github.com/user-attachments/assets/d11dc2c2-dc4e-4e87-8dd9-4a92ce033ae1" />

**Reply by Taiizor** [reply]
I’m not planning to add live wallpaper support for lock screens, as this could technically cause critical errors for end users.

I have considered a widgets feature for Sucrose, but I’m unsure if this would really fit the app. It could perhaps be developed as a separate application, but given that apps like [Rainmeter](https://www.rainmeter.net) already exist, I haven’t focused much on it.

<img width="900" height="600" alt="image" src="https://github.com/user-attachments/assets/9b99967c-cd12-451b-ad26-b545114c628d" />

Sucrose is released in different versions through different update channels. Since you're using the default channel, you receive the standard updates. The developer channel, on the other hand, includes more unstable versions. When a wallpaper is published in the store, it may require the latest version of Sucrose.

Your application might be up to date, but a newer version could have been released on the developer channel. For example, while version 25.9.19.0 is available on the default channel, version 25.10.15.0 has been released on the developer channel.

You can watch a guide about this here: https://www.youtube.com/watch?v=9N5RqjqgPSM

**Comment by Yin-Vannthy** [comment]
@Taiizor 
I just got this error when I tried to update to the latest version

<img width="583" height="294" alt="image" src="https://github.com/user-attachments/assets/774e310e-8692-4c36-87be-346f77943bc7" />

**Reply by Taiizor** [reply]
Ah yes, I believe I’ve fixed this issue in the latest version. But for now, if you click the `Solve Problem` button, it will start downloading the new version in your browser.

**Reply by Taiizor** [reply]
Or you can make the selection below.

<img width="900" height="600" alt="image" src="https://github.com/user-attachments/assets/e2ddf62e-9ba2-4111-8939-cdae504b41f0" />

**Comment by Yin-Vannthy** [comment]
@Taiizor 
Excuse me
I appreciate your hard work.
Is there a feature that automatically pauses video playback when a window is maximized or enters full-screen mode?

Thank you.

**Reply by Taiizor** [reply]
Ah, of course — there are plenty of similar features available. You just need to check the `Performance Settings` section of the app (:

<img width="900" height="600" alt="image" src="https://github.com/user-attachments/assets/628213a4-6a62-4aa8-89dc-2e2d1c074531" />
